examples/welcome.cr

```diff
@@ -7,11 +7,25 @@ client = Discord::Client.new(token: "Bot MjI5NDU5NjgxOTU1NjUyMzM3.Cpnz31.GQ7K9xw
 cache = Discord::Cache.new(client)
 client.cache = cache
 
+welcome_channel = 0_u64
+
+client.on_message_create do |payload|
+  # Here we take for example "#welcome", "#general" or some other channel as the argument and then we put the ID of it into welcome_channel.
+  if payload.content.starts_with? "!setwelcome"
+    welcome_channel = payload.content.split(" ")[1][2..-2].to_u64
```

Comment:
This is pretty unsafe. This should probably done with regex and return an error to the user if the argument is invalid.

Reply:
Yes I know thats what I meant with "I know that this bot is still improvable (for example that he doesn't takes everything as an argument at !setwelcome) but I want it to be simple and not too complex.". I thought first you could maybe leave that to others because so they learn more. Because im very sure the most people who look at these examples are beginners. But okay I think I will change that.

Reply:
Validating user input is a core concept to making chat bots. Doing something simple with regex should be something everyone should be familiar with, and still digestible for beginners, granted, this library intentionally is not aimed at beginners to Discord's API.

But yeah, please do this. This is way too error prone, and if users mess up the input it will only make them more confused trying to address exceptions from cases that aren't covered here.